docs(bus): improve `HttpComandDispatcher` description (#24)
NeuraLegion/secbox-sdk-js: packages/bus/README.md

For more information, please see `@secbox/core`.

> ### HttpCommandDispatcher

Reviewer: Remove heading, the test below should be a part of `Executing RPC methods` section

> The `HttpCommandDispatcher` is an alternative way to execute RPC command. To use `HttpCommandDispatcher` pass the following options object to the constructor method:

Reviewer: ```suggestion
The `HttpCommandDispatcher` is an alternative way to execute the commands over HTTP. To start, you should create an `HttpCommandDispatcher` instance by passing the following options to the constructor:
```

```ts
import { HttpCommandDispatcher, HttpCommandDispatcherConfig } from '@secbox/bus';

const options: HttpCommandDispatcherConfig = {

>   baseUrl,

Reviewer: `baseUrl` is not defined
```suggestion
  baseUrl: 'https://app.neuralegion.com',
```

  token: 'weobbz5.nexa.vennegtzr2h7urpxgtksetz2kwppdgj0',
  timeout: 10000,
  rate: {
    limit: 3,
    window: 2

>   }

Reviewer: It is better to omit any extra options in the examples. You should use only these which are necessary to start working with an entity.
```suggestion
```

};
  
const httpDispatcher = new HttpCommandDispatcher(options);
```

> The `HttpCommandDispatcherConfig` implementation exposes the properties described below:

Reviewer: `HttpCommandDispatcherConfig` is interface, not an implementation.

```suggestion
The command dispatcher can be customized using the following options:
```

| Option    | Description                                                                                                                                          |
|-----------|------------------------------------------------------------------------------------------------------------------------------------------------------|

> | `baseUrl` | Application url                                                                                                                                      |

Reviewer: ```suggestion
| `baseUrl` |  Base URL for your application instance, e.g. `https://app.neuralegion.com`                                                                          |
```

> | `token`   | Authorization token                                                                                                                                  |

Reviewer: ```suggestion
| `token`   | API key to access the API. Find out how to obtain [personal](https://docs.brightsec.com/docs/manage-your-personal-account#manage-your-personal-api-keys-authentication-tokens) and [organization](https://docs.brightsec.com/docs/manage-your-organization#manage-organization-apicli-authentication-tokens) API keys in the knowledgebase                                                                                                                                  |
```

> | `timeout` | If the request takes longer than `timeout`, the request will be aborted. Default 10000                                                               |

Reviewer: ```suggestion
| `timeout` | Time to wait for a server to send response headers (and start the response body) before aborting the request. Default 10000 ms                                                               |
```

> | `rate`    | Contain two options: `window` - amount of time to limit concurrent requests; `limit` - max requests to perform concurrently in given amount of time. |

Reviewer: ```suggestion
| `rate`    | Set how many requests per interval should perform immediately, others will be delayed automatically. By default, 10 requests per 1 minute |
```

> To execute command `HttpComandDispatcher` exposes a `execute()` method. This method is intended to perform a command to the application and returns a `Promise` with its response.

Reviewer: There is no point to repeat the section above, you should focus on the difference between a regular command and HTTP command. So, you can remove the text below entirely.

````suggestion
Then you have to create an instance of `HttpRequest` instead of a custom command, specifying the `url` and `method` in addition to the `payload` that a command accepts by default:

```ts
const command = new HttpCommand({ 
  url: '/api/v1/repeaters', 
  method: 'POST', 
  payload: { name: 'test' } 
});
```

Below you will find a list of parameters that can be used to configure a command:

--- PUT THE TABLE HERE --- 

Once it is done, you can perform a request using `HttpComandDispatcher` as follows:
```ts
const response: { id: string } = await httpDispatcher.execute(command);
```
````

| `params`        | Query parameters                                                                           |

> | `createdAt`     | The exact date and time the command was created.                                           |

Reviewer: Please sort params accordingly: own props first, then inherited props. 

Moreover, descriptions should be updated:

* `url` - Set an absolute URL or path that will be used for the request.
* `method` - Set an HTTP method that is going to be used when making the request.
* `params` - Use to set query parameters.

#### Custom HttpComandDispatcher

You can implement your own `HttpCommandDispatcher`. To do it you should implement `ComandDispatcher` interface.

```ts
import { Command, ComandDispatcher } from '@secbox/core';

export class CustomHttpDispather implements ComandDispatcher {
  constuctor(/*options*/) {
    // ...
  }
  

>   publick execute<T, R>(command: Command<T, R>): Promise<R> {

Reviewer 2: typo

>     const { url, method, payload } = command;

Reviewer 2: `url` & `method`  are `HttpRequest` (`Command` descendant) specific

  
    // implementation based on your http client
  }
}
```

> For more information, please see @secbox/core.

Reviewer: Please remove this section
